pom.xml

```diff
@@ -91,6 +91,7 @@
         <guava.version>20.0</guava.version>
         <parquet-protobuf.version>1.8.1</parquet-protobuf.version>
         <protobuf-dynamic.version>0.9.4</protobuf-dynamic.version>
+        <presto.version>0.206</presto.version>
```

Comment:
does this mean on every presto release we need to also bump and rollout Garmadon?

Reply:
No unless presto change is classloading mechanism + classes associated to it.